utilities/validations.py

```diff
@@ -133,6 +133,14 @@ def validate_structure_id(**kwargs):
     table
     :param structure_id
     """
+    if(kwargs.get('check_unique_structure_id')):
+        for structure in kwargs['data']:
+            existing_structure = StructureModel.query.filter_by(
```

Comment:
@jesseinit You might want to avoid using a verb `check_unique_structure_id` as a variable name

Reply:
Also, kindly explain why the flag `check_unique_structure_id` is necessary.

Reply:
@rajeman Thanks for the feedback, can you please kindly suggest a better variable name as I felt that variable best describes what it's doing as a parameter.
Also, the variable is sort of a settings parameter that extends the functionality of the `validate_structure_id` function. So when the variable is set to `True` it simply queries the StructureModel, checks for existing structures and if any structure is found raises an exception.

Reply:
I think I understand where Raji is coming from. Ideally, variables should be named with nouns and functions/methods with verbs. How about just removing the 'check'. What do you think?

Reply:
@rajeman @Makwe-O Updated